src/app/shared/player/player.component.ts

```diff
@@ -95,11 +95,15 @@ export class PlayerComponent implements OnInit, OnChanges {
     });
 
     this.player.addEventListener('playing', (e: Event) => {
-      this.play.emit(e);
+      if (!this.isScrubbing) {
+        this.play.emit(e);
+      }
     });
 
     this.player.addEventListener('pause', (e: Event) => {
-      this.pause.emit(e);
+      if (!this.isScrubbing) {
+        this.pause.emit(e);
```

Comment:
Issue #2: Don't emit the `pause` event while seeking, because that will trigger the overlay, which in turn blocks seeking.